src/fmt_md.rs

```diff
@@ -1830,9 +1833,10 @@ pub mod tests {
         /// see [crate::footnote_transform::test] for more extensive tests TODO need to add those tests!
         #[test]
         fn footnote_transform_smoke_test() {
-            check_render_refs_with(
+            let (ctx, graf) = footnote_a_in_paragraph();
+            check_render_refs_with_ctx(
                 &MdOptions::new_with(|mdo| mdo.inline_options.renumber_footnotes = true),
-                vec![MdElemRef::Paragraph(&footnote_a_in_paragraph())],
+                (ctx, vec![MdElemRef::Paragraph(&graf)]),
```

Comment:
🤏 "graf" is your journalism background sneaking into your code ;)

```suggestion
                (ctx, vec![MdElemRef::Paragraph(&paragraph)]),
```